SUPPORT.md

```diff
@@ -1,15 +1,31 @@
-# 😍 Support for deploying and using Catppuccin!
+<div align="center">
+  <h2>🫶 Support</h2>
+  <p>Where to get help in case things don't work</p>
+</div>
 
-Welcome to Catppuccin! We use GitHub for tracking bugs and feature requests. This isn't the right place to get support for using Catppuccin, but the following resources are available below, thanks for understanding.
+Welcome to Catppuccin! We have many ports across the organisation contributed
+and maintained by people all over the world. However, issues and bug reports are
+never-ending due to how many we have!
 
-## Real-time Chat
+If you are encountering an issue with a port, follow the guidance listed below:
 
-- [Discord](https://discord.com/servers/catppuccin-907385605422448742): The Catppuccin Community is active on Discord, you can post your questions there.
+1. **Make sure you have read and followed the `README`**. Every port will have this
+   file, explaining how to install the theme or modify the application to install
+   Catppuccin. It is shown by default when you visit a GitHub repository.
+2. **Raise a GitHub Issue**. If you have followed the `README` and are still
+   having trouble applying Catppuccin, raise a GitHub issue clearly explaining what
+   you have tried and what you expect to happen. It's also recommended to provide
+   extra details like your operating system, what version of the theme, etc. (if
+   relevant!)
```

Comment:
Worth mentioning that said GitHub issue should be raised on the appropriate/related repository